common/plugins/eu.esdihumboldt.hale.common.align/src/eu/esdihumboldt/hale/common/align/transformation/function/impl/PropertyValueImpl.java

```diff
@@ -69,6 +71,14 @@ public <T> T getValueAs(Class<T> type) throws ConversionException {
 			return (T) value;
 		}
 
+		if (value instanceof InstanceDecorator) {
+			return (T) ((InstanceDecorator) value).getValue();
+		}
+		
+		if (value instanceof DefaultInstance) {
+			return (T) ((DefaultInstance) value).getValue();
+		}
```

Comment:
Please replace both cases by once case checking for `instanceof` related to the `Instance` interface.